### DISCUSSION TITLE
SLES12 strongswan upgrade

### DISCUSSION THREAD

**Discussion by sridharkondeti** [strongswan/strongswan#1631] [General]
Hi,
Looks like the latest strongswan version that Suse Linux 12 supports is 5.1.3, but we are having some issues with this version. Is there any way we can install latest strongswan version on sles12 instances?

**Comment by tobiasbrunner** [comment]
Possibly from sources, see [here](https://docs.strongswan.org/docs/5.9/install/install.html#_compile_strongswan_yourself).
